File: Assignments/Solutions/Shivam Arora/ASSIGMENT No.1PATTERNS/Question-1.cpp
/*Question 1.

             *  *  *  *  *
          *  *  *  *  *
       *  *  *  *  *
    *  *  *  *  *
 *  *  *  *  *

*/

#include<iostream>
using namespace std;

int main(){
int n;
cin>>n;
for(int i=1;i<=n;i++){
    for(int j=i;j<=n-1;j++){
            cout<<" ";
    }
    for(int k=1;k<=n;k++){
            cout<<"*";
    }
    cout<<endl;
}
return 0;
}

File: Assignments/Solutions/Nimisha Agarwal/Assignment No.1/Question 1.py
"""
Question 1.
             *  *  *  *  *
          *  *  *  *  *
       *  *  *  *  *
    *  *  *  *  *
 *  *  *  *  *
"""

for i in range(1,6):
    for j in range(5,i,-1):
        print(" ",end="  ")
    for k in range(1,6):
        print("*",end="  ")
    print(end="\n")
